ui/js/class.FormInput.js

```diff
@@ -100,7 +101,12 @@ class FormInput extends $.Component {
 			for (const child of this.children) {
 				child.toggle(isEditable)
 			}
-			this.inPlaceLabel.props.text = this.stringValue || '-'
+			this.inPlaceLabel.props.text = this.stringValue || this.props.overlayPlaceholder
```

Comment:
Rename "overlayPlaceholder" to "defaultValue". Also need cast to String this.props.defaultValue